packages/core/demo/index.ts

```diff
@@ -150,57 +142,25 @@ const createChartContainer = chartType => {
 };
 
 // Initialize all charts
-chartTypes.forEach(type => {
-	const holder = createChartContainer(type);
-	if (holder) {
-		let classToInitialize;
-		switch (type.id) {
-			case "simple-bar":
-			case "simple-horizontal-bar":
-			case "simple-bar-time-series":
-			case "simple-horizontal-bar-time-series":
-				classToInitialize = SimpleBarChart;
-				break;
-			case "grouped-bar":
-			case "grouped-horizontal-bar":
-				classToInitialize = GroupedBarChart;
-				break;
-			case "stacked-bar":
-			case "stacked-horizontal-bar":
-			case "stacked-bar-time-series":
-			case "stacked-horizontal-bar-time-series":
-				classToInitialize = StackedBarChart;
-				break;
-			case "scatter":
-			case "scatter-time-series":
-				classToInitialize = ScatterChart;
-				break;
-			case "line":
-			case "line-time-series":
-			case "line-step":
-			case "line-step-time-series":
-				classToInitialize = LineChart;
-				break;
-			case "pie":
-				classToInitialize = PieChart;
-				break;
-			case "donut":
-				classToInitialize = DonutChart;
-				break;
+demoGroups.forEach(demoGroup => {
```

Comment:
Woohoo! way more readable than the crazy switch statement 😁

Reply:
That was part of the _original_ codebase 🤣